account_payment_promissory_note/wizard/account_register_payments.py

```diff
@@ -21,8 +22,8 @@ def _onchange_promissory_note(self):
         super()._onchange_promissory_note()
         if not self.date_due and self.promissory_note:
             invoices = False
-            if self._name == "account.register.payments":
+            if self._name == "account.payment.register":
```

Comment:
This is not needed (even in v12, I missed this one...). The onchange is already in the proper model this `if` is going to be always True.